### DISCUSSION TITLE
Is it possible to open tiddlywiki files in wikipad?

### DISCUSSION THREAD

**Discussion by ghost** [WikidPad/WikidPad#85] [Q&A] [answered]
Hi.

I would like to know if it is possible to open tiddlywiki files in wikipad, if there is a level of compatibility in these two programs: tiddlywiki and wikipad. *So... Is it possible to open tiddlywiki files in wikipad?*

**Comment by cknoll** [comment, marked as answer]
I think this is not possible. ASFAIK tddlywiki uses completely different technical approach (e.g. HTML+Javascript instead of SQLite database + python).
